openslides_backend/action/actions/projector_countdown/create.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Dict
+
+from ....models.models import ProjectorCountdown
+from ....shared.exceptions import ActionException
+from ....shared.filters import And, FilterOperator
+from ....shared.patterns import Collection, FullQualifiedId
+from ...generics.create import CreateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+
+
+@register_action("projector_countdown.create")
+class ProjectorCountdownCreate(CreateAction):
+    """
+    Action to create a projector countdown.
+    """
+
+    model = ProjectorCountdown()
+    schema = DefaultSchema(ProjectorCountdown()).get_create_schema(
+        required_properties=["meeting_id", "title"],
+        optional_properties=["description", "default_time"],
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        self.check_title_unique(instance)
+
+        # set default_time if needed and countdown_time
+        if not instance.get("default_time"):
+            meeting = self.datastore.get(
+                FullQualifiedId(Collection("meeting"), instance["meeting_id"]),
+                ["projector_countdown_default_time"],
+            )
+            instance["default_time"] = meeting.get("projector_countdown_default_time")
```

Comment:
What happens if `projector_countdown_default_time` is not set? Or should this field be required in the meeting?

Reply:
is should be required

Reply:
ok, I will open an issue for it.